### PR TITLE
Improve notifications configuration page

### PR DESCRIPTION
**Pull request by mdeniz**
This is how is actually looks like:

imagen

This is how it will look like:

Top Part:

imagen

Bottom part:

imagen

### CODE REVIEW

File: src/api/app/models/event/base.rb [hunk 1]
```diff
@@ -9,6 +9,17 @@ class Base < ApplicationRecord
 
     before_save :shorten_payload_if_necessary
 
+    EXPLANATION_FOR_NOTIFICATIONS =  {
+      'Event::BuildFail'          => 'Receive notifications of build failures for packages which you are involved in',
```
Comment (hennevogel) [inline src/api/app/models/event/base.rb]: Receive notifications of build failures for packages which you are involved in **as**

**Comment by hennevogel** [comment]
I'm also not sure that the 'Watcher' things make sense to any user.

**Comment by mdeniz** [comment]
Ok, then adding the `as` and removing the watching part...

**Comment by hennevogel** [comment]
hm what is the watching part? :-)

**Comment by mdeniz** [comment]
Ok, then changing the explanations to something like this:
> Receive notifications of requests state changes for projects which you are...

And then I will change the `Source Watcher` and `Target Watcher` to something more understandable.

**Comment by mdeniz** [comment]
@hennevogel @evanrolfe please re-review. I changed the literals. You can see the new screenshots too.

**Comment by hennevogel** [comment]
> You can see the new screenshots too.

Liar liar pants on fire! 🔥

File: src/api/app/models/event/base.rb [hunk 2]
```diff
@@ -9,6 +9,17 @@ class Base < ApplicationRecord
 
     before_save :shorten_payload_if_necessary
 
+    EXPLANATION_FOR_NOTIFICATIONS =  {
+      'Event::BuildFail'          => 'Receive notifications of build failures for packages which you are...',
```
Comment (hennevogel) [inline src/api/app/models/event/base.rb]: Receive notifications of build failures for packages **for** which you are...

Comment (mdeniz) [reply src/api/app/models/event/base.rb]: Ok, done

**Comment by hennevogel** [comment]
Awesome work 🍔  :-)

**Pull request merged**